etl/models/models.go

```diff
@@ -59,6 +72,45 @@ type JetDrop struct {
 	RecordAmount   int
 }
 
+type State struct {
+	RecordRef    []byte `gorm:"primary_key;auto_increment:false"` // State reference.
+	Type         StatesType
+	RequestRef   []byte // Reference to the corresponding request.
+	ParentRef    []byte // Reference to the parent object that caused creation of the given object.
+	ObjectRef    []byte
+	PrevStateRef []byte // Reference to a previous state.
+	IsPrototype  bool
+	Payload      []byte
+	ImageRef     []byte
+	Hash         []byte
+	Order        int
+	JetID        string
+	PulseNumber  int64
+	Timestamp    int64
+}
+
+type Request struct {
+	RecordRef          []byte // Request reference.
+	Type               RequestsType
+	CallType           string
+	ObjectRef          []byte // Reference to the corresponding object.
+	CallerObjectRef    []byte // Reference to the object that called this request.
+	CalleeObjectRef    []byte
+	APIRequestID       string // Internal debugging information,filled in case of working with v1 platform
+	ReasonRequestRef   []byte // Reference to the parent request—a request that caused this one
+	OriginalRequestRef []byte // original request, filled in case of working with v2 platform
+	Method             string // Name of the smart contract method that called this request.
+	Arguments          []byte // Arguments of a smart contract method.
+	Immutable          bool   // True if request didn't change the object state. False otherwise.
+	IsOriginalRequest  bool
+	PrototypeRef       []byte
+	Hash               []byte
+	JetID              string
+	PulseNumber        int64
+	Order              int
+	Timestamp          int64
```

Comment:
and here also

Reply:
It is a database type, why do you think it should be used `Reference`

Reply:
sorry ((